lib/Github/Api/Organization.php

```diff
@@ -53,14 +53,18 @@ public function update($organization, array $params)
      * @param string $organization the user name
      * @param string $type         the type of repositories
      * @param int    $page         the page
+     * @param string $sort         sort by
+     * @param string $direction    direction of sort, asc or desc
      *
      * @return array the repositories
      */
-    public function repositories($organization, $type = 'all', $page = 1)
+    public function repositories($organization, $type = 'all', $page = 1, $sort = 'created', $direction = 'desc')
```

Comment:
I would change this so the default value is `null`. Then check if the parameter is different from `null` and then add it to a `$parameters` array and pass it to the request.

This way we always use the default value of github and it won't cause BC breaks when github decided to change this default value

Reply:
Alright, I updated my PR. 😄